packages/server/src/sdk/app/appMetadata/metadata.ts

```diff
@@ -0,0 +1,18 @@
+import { context, DocumentType } from "@budibase/backend-core"
+import { App } from "@budibase/types"
+
+/**
+ * @deprecated the plan is to get everything using `tryGet` instead, then rename
+ * `tryGet` to `get`.
+ */
+export async function get() {
```

Comment:
There is an `application` layer to the SDK - would it not be better to add this there?

Reply:
Not sure if that is exactly the same. I was actually doubting where to put that. I understood that the current application is for the crud of the apps itself, but this is for the app metadata specific. Do you think it's best to merge both?

Reply:
I think it probably makes sense to have metadata functions under the application, the CRUD of the application does fall under that part of the SDK, but the metadata is pretty much the only application document there is and I would default to looking for it under that part of the SDK.